docs/book/src/topics/crd-changes/v1alpha6-to-v1alpha7.md

```diff
@@ -175,6 +175,28 @@ securityGroupFilters:
 - id: 4a131d3e-9939-4a6b-adea-788a2e89fcd8
 ```
 
+### Change to ports.bindingProfile
+
+We previously allowed to use the Neutron `binding:profile` via `ports.bindingProfile` but this API is not declared as stable from the [Neutron API description](https://docs.openstack.org/api-ref/network/v2/index.html?expanded=create-port-detail#create-port).
```

Comment:
It used to be called `port.profile`, not `ports.bindingProfile`. If we do decide to change the name we should document that clearly here.

We should also be explicit that the conversion is lossy, and if anyone was relying on anything other than the supported behaviour it will be lost.